docs/topics/extensions.rst

```diff
@@ -247,39 +247,25 @@ Spider state extension
 ~~~~~~~~~~~~~~~~~~~~~~
 
 .. module:: scrapy.extensions.spiderstate
-   :synopsis: Close spider extension
+   :synopsis: Spider state extension
 
 .. class:: SpiderState
 
-Stores and loads spider state during a scraping job. It is especially useful when resuming a paused or interrupted job, as it ensures
-that the spider can retain its internal state, allowing for a more seamless continuation of the scraping process.
-
-.. note:: To enable `SpiderState`, ensure that the `JOBDIR` setting is configured in your Scrapy project. Without this setting, the extension will raise a `NotConfigured` exception and will not be activated.
-
-The extension is used while:
-
-- **Opening a Spider**: When a spider is opened, the extension checks if a previous state file exists in the specified `JOBDIR`.
-  If found, it loads the state from the file and assigns it to `spider.state`.
-- **Closing a Spider**: When a spider closes, the extension saves the `spider.state` attribute to a file within `JOBDIR`.
-  This stored state can be reloaded in subsequent sessions, enabling continuity.
-
-
-Implementation of state management in a spider:
-
-.. code-block:: python
-
-    from scrapy import Spider
+Stores and loads spider state during a scraping job.
```

Comment:
```suggestion
Stores and loads spider state data during a scraping job.
```

“during a scraping job” feel misleading to me. It loads the spider state data *before* the crawl starts, and stores it *after* the crawl stops, it does not do anything during the crawl.

Reply:
How does this look for the intro "Manages spider state data by loading it before a crawl and saving it after."